app/scripts/modules/appengine/pipeline/stages/enableAsg/appengineEnableAsgStage.ts

```diff
@@ -0,0 +1,49 @@
+import {module} from 'angular';
+
+import {IAppengineStageScope} from 'appengine/domain/index';
+import {ACCOUNT_SERVICE, AccountService} from 'core/account/account.service';
+import {AppengineStageCtrl} from '../appengineStage.controller';
+
+class AppengineEnableAsgStageCtrl extends AppengineStageCtrl {
+  static get $inject() { return ['$scope', 'accountService']; }
+
+  constructor(public $scope: IAppengineStageScope, accountService: AccountService) {
```

Comment:
Please correct me if I'm wrong, but I think they should be `protected` -- I'm calling a base class below, and the compiler complains if these are `private`, regardless of the access modifiers on the base class.